Mage/src/main/java/mage/game/GameImpl.java

```diff
@@ -145,6 +149,8 @@
     // temporary store for income concede commands, don't copy
     private final LinkedList<UUID> concedingPlayers = new LinkedList<>();
 
+    private Map<UUID, FilterCreaturePermanent> usePowerInsteadOfToughnessForDamageLethalityFilters = new HashMap<>();
```

Comment:
`GameImpl` must contains only global and non game state data. All gameplay and changeable data must be stored in `GameState` object (rollback restores GameState, but not GameImpl).

Reply:
ah, gotcha, makes sense

Reply:
fixed